src/main/java/frc/robot/subsystems/climber/ClimberSS.java

```diff
@@ -0,0 +1,47 @@
+package frc.robot.subsystems.climber;
+
+import edu.wpi.first.math.MathUtil;
+import frc.robot.constants.RobotConstants.ClimberConstants;
+import frc.robot.subsystems.OverridableSubsystem;
+import frc.robot.utilities.pid.PIDFTalonFX;
+
+public class ClimberSS extends OverridableSubsystem {
+    private final PIDFTalonFX leftMotor;
+    private final PIDFTalonFX rightMotor;
+
+    public ClimberSS() {
+        leftMotor = ClimberConstants.LEFT_MOTOR;
+        rightMotor = ClimberConstants.RIGHT_MOTOR;
+    }
+
+    public void resetEncoder() {
+        leftMotor.setSelectedSensorPosition(0);
+        rightMotor.getSelectedSensorPosition(0);
+    }
```

Comment:
1. put this below get
    public double getAveragePosition() 
2. call this in constructor

Reply:
I did not understand..

Reply:
I did understand..